### PR TITLE
Add quote-of-the-day-dotnet template

### PR DESCRIPTION
**Pull request by amerjusupovic**
### Your template repository
Place your template repository link here: https://github.com/Azure-Samples/quote-of-the-day-dotnet

(No PR link for when azd compatibility was added since it was originally designed to use azd)

- [x] Added an entry to https://github.com/Azure/awesome-azd/blob/main/website/src/data/users.tsx that includes:
    - [x] **Template title** - A short title that reflects the local application stack that someone could use to get their application on Azure (e.g. "Containerized React Web App with Java API and MongoDB")
    - [x] **Description** - 1-2 sentence description of the architecture (e.g. Azure services) or solution that is defined by the template.
    - [x] **Architecture Diagram or Application Screenshot** - The image should include all services and their connections ([example](https://github.com/Azure-Samples/todo-csharp-sql/blob/main/assets/resources.png)). You should add the image to the `website/src/data/images/`.
    - [x] **Link to Author's GitHub or other relevant website** - Used for attribution
    - [x] **Author's Name** -  Name to credit on the gallery card
    - [x] **Link to template source** - Link to the template GitHub repo 
    - [x] **Tags** - Specify [tags](https://github.com/Azure/awesome-azd/blob/main/website/src/data/tags.tsx) to represent the template. If you don't see a relevant tag for your template? Feel free to add one!
        
        **Required tags**:
      - [x] Tag your template as Micrsoft-authored (`"msft"`) or Community-authored (`"community"`)
      - [x] Tag the IaC provider (`"bicep"` or `"terraform"`)
      - [x] At least 1 tag for programming language used
      - [x] At least 1 tag for Azure services integrated
      - [x] Add the `"new"` tag for any newly authored templates

- [x] In the PR comment, if you can also add a link to the PR where you made your repo `azd` compatible this will allow us to provide feedback on your template and speed up the review process.
- [x] If the template is Microsoft-authored, we encourage you to also [publish it to learn.microsoft.com/samples](https://review.learn.microsoft.com/en-us/help/contribute/samples/process/onboarding?branch=main).

# If you are submitting a resource to be added to the awesome-azd README:
- [ ] Name of resource
- [ ] Resource author - who created this resource? (so we can credit them!)
- [ ] What section should this resource be included in? -Is the resource an article? A video? Something else?

### CODE REVIEW

**Comment by amerjusupovic** [comment]
Quick question: when users search for a template after calling `azd init`, what's the name my template would show up as? Is it the repo name?

**Comment by v-xuto** [comment]
> @v-xuto Could you test this template?

Testing

**Comment by v-xuto** [comment]
@hemarina, @amerjusupovic  We have finished this template test, and we filed one issue. Please review.
- https://github.com/Azure-Samples/quote-of-the-day-dotnet/issues/10

**Comment by amerjusupovic** [comment]
Thanks, looking into it

**Comment by amerjusupovic** [comment]
Merged PR with fixes

File: website/static/templates.json [hunk 1]
```diff
@@ -2152,5 +2150,24 @@
...
+    "website": "https://github.com/Azure/AppConfiguration",
+    "author": "Azure App Configuration",
+    "source": "https://github.com/Azure-Samples/quote-of-the-day-dotnet",
+    "tags": [
+      "msft",
+      "dotnetCsharp",
+      "appinsights",
+      "appservice",
+      "azureappconfig",
+      "experimentation",
```
Comment (hemarina) [inline website/static/templates.json]: `experimentation`, `featuremanagement`, and `split` are new tags. Could you share full name of the tag please?

**Comment by v-xuto** [comment]
@amerjusupovic We tested based on the latest code, and there are new errors when executing `azd up`. Please see the [comments ](https://github.com/Azure-Samples/quote-of-the-day-dotnet/issues/10#issuecomment-2339921745)for more details.

- https://github.com/Azure-Samples/quote-of-the-day-dotnet/issues/10 (Has new issue)

**Comment by amerjusupovic** [comment]
So sorry about that, I didn't run into this when testing last time. I'll look into it again, thanks

File: website/static/templates.json (hunk 1 above)
Comment (amerjusupovic) [reply website/static/templates.json]: My fault, I see where the other tags are defined now. Do I need to add them to the tags.tsx file? l will likely drop the `split` tag but I want to add the other two as "Microsoft Feature Management" and "Feature Experimentation".

**Comment by amerjusupovic** [comment]
Updated the linked issue in the quote of the day repo

File: website/static/templates.json (hunk 1 above)
Comment (hemarina) [reply website/static/templates.json]: Thank you for your patient. Could you add those two tags? I couldn't directly push to your branch due to permission deny.

File: website/static/templates.json [hunk 2]
```diff
@@ -2152,5 +2150,24 @@
...
+      "bicep",
+      "featuremanagement",
+      "split"
```
Comment (hemarina) [inline website/static/templates.json]:
```suggestion
      "new"
```

File: website/static/templates.json (hunk 1 above)
Comment (hemarina) [inline website/static/templates.json]:
```suggestion
      "featureExperimentation",
```

**Review by gkulin — APPROVED** [review]

**Pull request merged**